Documentation of some changes

luigifab opened the pull request:
Here is a draft with the changes between OM 19.x and 20.x.
It's not very good but it's a start.

Sdfendor commented on README.md:
```diff
@@ -74,15 +73,60 @@ git add -A && git commit
+- removed mysql4 classes #1470
+- removed admin routing compatibility mode for extensions #1551
+
+Futur changes:
```
Futur => Future

luigifab replied:
+1

elidrissidev commented on README.md:
```diff
@@ -74,15 +73,60 @@ git add -A && git commit
+
+### Between OpenMage 20.x and 21.x
+
+Do not use 21.x.x if you don't known what do you do, or if you don't understand consequences of the following changes.
```
```suggestion
Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.
```

elidrissidev commented on README.md:
```diff
@@ -74,15 +73,40 @@ git add -A && git commit
+- add redis as a valid option for `global/session_save` #1513
+- possibility to disable global search in backend #1532
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v19.4.15...v20.0.13).
```
```suggestion
For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/1.9.4.x...20.0).
```

elidrissidev commented on README.md:
```diff
@@ -74,15 +73,40 @@ git add -A && git commit
+- removed mysql4 classes #1470
+- removed admin routing compatibility mode for extensions #1551
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v20.0.13...v21.0.0).
```
```suggestion
For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/20.0...21.0).
```
Also, why are we documenting a version that doesn't exist yet?

luigifab replied:
Because I hope it will be released.
In UTC time there are still some hours before 2022!

elidrissidev replied:
Well yeah, I would prefer if it aligns with year i.e 22 instead of 21

ADDISON74 commented:
Except the part which refers to OpenMage version 21 everything looks good to me.

tmotyl commented:
I also don't know what is the story with v21?

fballiano commented:
just to explain: at that time there was a discussion about the removal of the mysql classes and maybe move it a another major release called v21. then the whole thing was paused.

luigifab commented:
For the History, I like my breaking ideas!

```md
### Between OpenMage 20.x and 21.x

Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.

- removed mysql4 classes
- removed admin routing compatibility mode for extensions

For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/20.0...21.0).
```

elidrissidev commented:
@fballiano Can we keep using squash and merge instead of a merge commit? It makes the history cleaner and easier to read through.

fballiano commented:
> @fballiano Can we keep using squash and merge instead of a merge commit? It makes the history cleaner and easier to read through.

I'll take a look into that! Thanks for the suggestion, it actually makes a lot of sense!
